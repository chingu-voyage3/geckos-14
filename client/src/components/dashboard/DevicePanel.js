import React from 'react';
import DeviceList from './DeviceList';
import ControlPanel from './ControlPanel';

const DevicePanel = props => {
  // console.log(props);
  return (
    <div className="device-panel">
      <ControlPanel params={props.params} actions={props.actions} parent="device" />
      <DeviceList devices={props.devices} />
    </div>
  );
};

export default DevicePanel;
