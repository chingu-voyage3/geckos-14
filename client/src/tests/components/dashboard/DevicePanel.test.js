import React from 'react';
import DevicePanel from '../../../components/dashboard/DevicePanel';

import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const div = document.createElement('div');
  const wrDevicePaneler = shallow(<DevicePanel />);
  expect(toJson(wrDevicePaneler)).toMatchSnapshot();
});
