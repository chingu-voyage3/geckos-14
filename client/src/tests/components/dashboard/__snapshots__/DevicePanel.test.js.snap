// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="device-panel"
>
  <ControlPanel
    parent="device"
  />
  <DeviceList />
</div>
`;
